README.md

```diff
@@ -23,21 +23,36 @@ Current reviewers are listed in [`bioimageio_collection_config.json`][review-con
    * request changes on resource drafts
    * upload a new version for any resource
 
-### Review Process
+### Adding a bioimage.io resource
 
-The review process technically starts after a user uploaded a _resource package_.
-Such a _resource package_ could e.g. be a newly uploaded _model package_, or _notebook package_, or an updated version of any existing resource.
+A bioimage.io resource is created by a user uploading a _resource package_.
+Such a _resource package_ could e.g. be a newly uploaded _model package_, or _notebook package_.
```

Comment:
so the user is uploading a _resource package_, but the _resource package_ is already a newly uploaded _model package_, so uploading twice? ;)

Reply:
no any _... package_ is just a zip file...  ?